jdaviz/core/loaders/test_loaders.py

```diff
@@ -189,12 +189,8 @@ def test_image_mult_sci_extension(imviz_helper):
                         fits.ImageHDU(arr, name='ERR', ver=2)
                          ])
 
-    ldr = imviz_helper.loaders['object']
-    ldr.object = hdul
-
-    ldr.importer.extension = ['SCI,1', 'SCI,2', 'ERR,2']
-    assert len(ldr.importer.extension.selected) == 3
-    ldr.importer()
+    # imviz_helper._load(hdul, extension=('SCI,1', 'SCI,2', 'ERR,2'))
+    imviz_helper.load_data(hdul, ext=('SCI,1', 'SCI,2', 'ERR,2'))
```

Comment:
this test `load` via `load_data` so that we also know that the mapping there is handled, once `load_data` is gone, we can replace with the call above that uses `extension` in place of the old kwarg `ext`